workflows: fix description of backoff coefficient (#4402)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

| --- | --- |
| **Maximum number of attempts** | The maximum number of times to execute the activity or child workflow. |

> | **First retry interval** | The amount of time to wait before the first retry. |

Reviewer: Given that we are in this doc, can you add the time units for all the rows in this table. For example is this in seconds, minutes etc.

Author: Conceptually doesnt make sense to document here the time units for this as it will depend on the SDK implementation, each language has its own way of expressing time durations, i.e in golang its configured with `10*time.Millisecond` or `1*time.Hour`

> | **Backoff coefficient** | The coefficient used to determine rate of increase of back-off, i.e a coefficient of 2 will double the wait of each subsequent retry. |

Reviewer: ```suggestion
| **Backoff coefficient** | The coefficient used to determine the rate of increase of back-off. For example a coefficient of 2 doubles the wait of each subsequent retry. |
```

| **Maximum retry interval** | The maximum amount of time to wait before each subsequent retry. |

> | **Retry timeout** | The overall timeout for retries, regardless of any configured max number of attempts. |

Reviewer: What unit of time is this?

Author: same response as before
